tests/manage/z_cluster/cluster_expansion/test_node_expansion.py

```diff
@@ -16,19 +15,25 @@ class TestAddNode(ManageTest):
     """
     Automates adding worker nodes to the cluster while IOs
     """
-
-    def test_add_node(self):
+    @aws_platform_required
```

Comment:
why do we drop vmware?

Reply:
gonna have separate function for vmware.